### PR TITLE
Add alert about site naming restrictions

### PR DESCRIPTION
**Pull request by stevector**
This PR is a modification to https://github.com/pantheon-systems/documentation/pull/9046.

When [describing site creation](https://docs.pantheon.io/guides/getstarted/addsite) we should specify the limits placed on names.

### CODE REVIEW

**Comment by stevector** [comment]
@stephpawlowski can you 👍 this PR?

**Comment by stephpawlowski** [comment]
looks good to me, very straightforward. is it worth listing that, site names cannot be changed once created?

File: source/content/guides/getstarted/06-addsite.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ To create a CMS site:
 
 1. Enter the name and select a region for this site. If this site is to be part of a Professional Workspace, select a Workspace from **Choose a Workspace for the Site**. Click **Continue**. It can take several minutes to create a new site on Pantheon.
 
+   <Alert title="Note" type="info">
+
+   Site names are limited to 51 characters and can contain only numbers, letters, and dashes. Site names cannot start or end with a dash.
+
```
Comment (stevector) [inline source/content/guides/getstarted/06-addsite.md:43]:
```suggestion

Site names cannot be changed after site creation.

```

**Comment by stevector** [comment]
Thanks @stephpawlowski! @rachelwhitton I added another sentence about site names can't be changed after creation. @rachelwhitton please review and merge.

**Pull request merged**